### DISCUSSION TITLE
Asistencia reuniones en vivo

### DISCUSSION THREAD

**Discussion by LuiyiBarrantes** [Launch-X-Latam/MisionFrontEnd#24] [Q&A] [answered]
Hace falta asistir a todas las sesiones en vivo?o hay que cumplir una asistencia? Porque en el horario que están fijadas, se me superpone el horario de trabajo, gracias.

**Comment by JosafatJimenezB** [comment, marked as answer]
no hay problema, están esta grabadas y  las podrás ver cuando puedas, pero recuerda no atrasarte compañero explorer! si se puede!!!

**Comment by Emilio17MC** [comment]
Como tal no se toma asistencia y las puedes ver grabadas, lo importante es que las veas, hagas las prácticas y participes en la comunidad

**Reply by gjordanra** [reply]
Hola como dijo hoy el commander es bueno que leas bien los correos y los post que van subiendo ahi encuentras la información sobre los requisitos para que puedas aprobar el bootcamp.

**Comment by rgplazas** [comment]
Gracias muy buena info comunidad
